### PR TITLE
Move diff page to the new design

### PR DESCRIPTION
**Pull request by baisa**
Move Compare with previous version page to GOV.UK Design System, using diffy gem for styling.

[Trello card](https://trello.com/c/mlEUeqal/891-compare-with-previous-version-diff-page)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

Before:

Screenshot 2022-11-24 at 10 34 02

After:

Screenshot 2022-11-23 at 17 13 22

### CODE REVIEW

**Comment by davidgisbey** [comment]
Code looks good to me outside of Daniels questions on the CSS 👍

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -14,6 +14,7 @@ $govuk-page-width: 1140px;
 @import "./admin/views/whats-new";
 @import "./admin/views/edit-edition";
 @import "./admin/views/attachments";
+@import "./admin/views/audit-trail";
```
Comment (kevindew) [inline app/assets/stylesheets/application.scss:17]: Just at a quick glance I notice we have a view CSS file here, but the CSS looks like it's global namespaced. 

I think we'll have to ponder whether this is a view's CSS or if it's a global module that can be used in the app.

This seems related to @dnkrj's comment.

File: app/assets/stylesheets/admin/views/_audit-trail.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
+// stylelint-disable max-nesting-depth
```
[the remark this answers is not in the document]
Comment (baisa) [reply app/assets/stylesheets/admin/views/_audit-trail.scss:1]: @dnkrj I have kept the same name for CSS as we had for legacy view. This CSS is taken from Travel Advice Publisher where we have already implemented this specific design.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (baisa) [reply app/assets/stylesheets/application.scss:17]: @dnkrj @kevindew I have scoped CSS with .audit-trail__history-comparison class

**Review by dnkrj — APPROVED** [review]

**Pull request merged**